packages/liveblocks-react-ui/src/utils/use-controllable-state.ts

```diff
@@ -2,11 +2,29 @@ import { console } from "@liveblocks/core";
 import { useCallback, useEffect, useRef, useState } from "react";
 
 export function useControllableState<T>(
+  /**
+   * The value used as an uncontrolled fallback if neither `value` nor `defaultValue` are provided.
+   */
+  fallbackValue: T,
+
+  /**
+   * The controlled value.
```

Comment:
Setting this to `undefined` makes it uncontrolled, right? That's the definition of whether something is controlled or uncontrolled? Could that be documented?

Reply:
Yes, I improved the comments a bit in https://github.com/liveblocks/liveblocks/pull/2449/commits/9e4fa022b9a95b69fa062e28c15f0d3e08c77304.